### ISSUE TITLE
Images are not displayed sometimes

### ISSUE THREAD

**Issue by Ingvord** [tango-controls/rest-server#78]
Originally reported by: **Ingvord (Bitbucket: [Ingvord](https://bitbucket.org/Ingvord), GitHub: [Ingvord](https://github.com/Ingvord))**

----------------------------------------

```
Image corrupt or truncated: data:/jpeg;base64,/9j/4AAQSkZJRgABAgAAAQABAAD...
```

----------------------------------------
- Bitbucket: https://bitbucket.org/hzgwpn/mtangorest.server/issue/78

**Comment by Ingvord** [comment]
*Original comment by* **Ingvord (Bitbucket: [Ingvord](https://bitbucket.org/Ingvord), GitHub: [Ingvord](https://github.com/Ingvord))**:

----------------------------------------

The problem seems to be in this line of code:

```
#!java
//Rest2Tango.java:370

if(ImageIO.write(image, "jpeg", out){
```

specifically when ImageIO writes a jpeg image it loses some bytes. This does not happen when it writes bmp image, but bmps are huge and using them is not appropriate for mTango. Maybe other writer will fix the problem...

**Comment by Ingvord** [comment]
*Original comment by* **Ingvord (Bitbucket: [Ingvord](https://bitbucket.org/Ingvord), GitHub: [Ingvord](https://github.com/Ingvord))**:

----------------------------------------

mTangoREST.server: fix #78 (???)

**Comment by Ingvord** [comment]
*Original comment by* **Ingvord (Bitbucket: [Ingvord](https://bitbucket.org/Ingvord), GitHub: [Ingvord](https://github.com/Ingvord))**:

----------------------------------------

Removing component: web-frontend (automated comment)
